beetmoverscript/src/beetmoverscript/task.py

```diff
@@ -30,6 +30,8 @@ def get_schema_key_by_action(context):
         return "import_from_gcs_to_artifact_registry_schema_file"
     elif utils.is_upload_data_action(action):
         return "upload_data_schema_file"
+    elif utils.is_upload_translations_artifacts_action(action):
+        return "upload_translations_artifacts"
```

Comment:
Should this action only be available for the translations cot product?  If not, 1) the name is going to be confusing, 2) what are the implications of that payload shape being allowed in, say, firefox?  (And now I think about it, the same applies to upload-data...)

Reply:
Yeah, making this exclusive to the translations production makes sense to me.

Reply:
Sigh...so we have no access to the cot product in the script itself (only the `scriptworker` process has it in the env/config). Perhaps this is guarded enough by the `upload-translations-artifacts` scope? I can wire this in further if you like though.